### PR TITLE
Removing sleeps

### PR DESCRIPTION
**Pull request by RyanLassigBanks**

### CODE REVIEW

File: integration/sawtooth_integration/tests/test_dynamic_network.py [hunk 1]
```diff
@@ -205,16 +205,12 @@ def stop_node(self, num):
...
-                return
-
-            time.sleep(sleep_time)
-            sleep_time *= growth_rate
+        timeout = ASSERT_CONSENSUS_TIMEOUT + time.time()
+        in_consensus = False
 
-        self.assertTrue(self.in_consensus())
+        while not in_consensus and time.time() < timeout:
+            in_consensus = self.in_consensus()
```
Comment (nick-drozd) [inline integration/sawtooth_integration/tests/test_dynamic_network.py]: `in_consensus` includes a call to `sawtooth block list`. If the validators aren't in consensus, how many times will `in_consensus` be called, and how frequently?

File: integration/sawtooth_integration/tests/test_poet_smoke.py [hunk 2]
```diff
@@ -85,17 +76,22 @@ def test_poet_smoke(self):
...
-            time.sleep(TRY_AGAIN_TIME)
-            self._assert_consensus()
+        time_out = time.time() + WAIT
+
+        while time.time() < time_out:
+            try:
+                self._assert_consensus()
+                return
+            except AssertionError:
+                LOGGER.info('Blocks not yet in concesus')
```
Comment (vaporos) [inline integration/sawtooth_integration/tests/test_poet_smoke.py]: Typo 'concesus' should be 'consensus'

Comment (vaporos) [inline integration/sawtooth_integration/tests/test_poet_smoke.py]: Suggest rewriting this loop so the logging message has more information (ideally, a list of the block ids at that point and the time which has passed).

File: integration/sawtooth_integration/tests/test_dynamic_network.py (hunk 1 above)
Comment (RyanLassigBanks) [reply integration/sawtooth_integration/tests/test_dynamic_network.py]: After running the tests a few times, it looks like in_consensus is called 1 to 6 times per call to assert_consensus. It doesn't seem to be affecting test stability, but we could add some sleeps back in to throttle it, if that's what you're worried about.

File: integration/sawtooth_integration/tests/test_poet_smoke.py (hunk 2 above)
Comment (RyanLassigBanks) [reply integration/sawtooth_integration/tests/test_poet_smoke.py]: Fixed type

Comment (RyanLassigBanks) [reply integration/sawtooth_integration/tests/test_poet_smoke.py]: added some logging to communicate how much time has passed. `_assert_consensus` already has logging to list the block ids

**Pull request merged**